CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 # Contributing to Jupyter Notebook
 
-Thanks for contributing to Jupyter Notebook!
+Thanks for showing interest on contributing to Jupyter Notebook!
```

Comment:
This sounds like bad grammar to me, there is only one result on Google for "Thanks for showing interest on contributing to".